### PR TITLE
adicionei as soluções propostas pelo professor Pedro Ruas

### PR DESCRIPTION
**Pull request by MatheusMotaGomes**
- [DEMO LINK](https://MatheusMotaGomes.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://MatheusMotaGomes.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Comment by MatheusMotaGomes** [comment]
Adicionei as mudanças propostas pelo mentor Pedro Ruas

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
```
Comment (jv-aquino) [inline src/style.css:55]: not wrong code, but it's a good idea to add a (or more) blank line between a big list of css properties (like 6+)

just a personal suggestion

File: src/index.html [hunk 2]
```diff
@@ -15,8 +15,73 @@
...
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          class="imagem"
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="links">
```
Comment (jv-aquino) [inline src/index.html:32]: for future BEM code (also if you wish to implement now): 
you should follow BEM structure:
links__link or something more semantic like nav__link

**Review by jv-aquino — CHANGES_REQUESTED** [review]
Good Matheus, but there's a problem with the UI replica - can you spot the typography difference? 

tip: font-weight

**Comment by MatheusMotaGomes** [comment]
Mudei o font-weight de 500 para normal, tentei outras variações porém os erros persistem. Por gentileza, só diga aonde estou errando kkkkk

**Pull request closed without merging**